src/kalite-app.py

```diff
@@ -42,5 +42,7 @@ def request_portal(url):
 sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
 connection = sock.connect_ex(('127.0.0.1', port))
 
-request_portal('http://localhost:8008')
+# We use a custom URI that will be handled by chromium-browser-appmode
+# so that chromium-browser gets launched with --class and -app set.
+request_portal('webapp://org.learningequality.KALite@http://localhost:8008')
```

Comment:
It's a bit tricky to check for an error here because the flow is something like this:

kalite-app (sandboxed) -> xdg-desktop-portal -> chromium-browser-appmode -> chromium-browser

Still, I think we should be able to at least know whether the portal succeeded to launch an application for the given scheme, and that's probably enough. I'll look into it